djangocms_rest_api/serializers/base.py

```diff
@@ -8,12 +8,27 @@
 from django.core.urlresolvers import reverse
```

Comment:
please change the imports to be consistent with:

1. Python
2. Django
3. djangCMS
4. third party
5. local